### PR TITLE
Mango/documentation

### PR DESCRIPTION
**Pull request by captainmangoC4**

### CODE REVIEW

**Comment by Simon-Busch** [comment]
Is it normal that the file register-juge has been deleted ?

**Comment by captainmangoC4** [comment]
@Simon-Busch yes, I noticed that `register-judge` was not used anywhere and was basically just a copy of `register-warden`. It seems it was replaced with `apply-for-judge`. This is probably another case of "should have done this in a separate PR" though. 🤪

File: README.md [hunk 1]
```diff
@@ -26,18 +26,52 @@ To register as a warden, follow the instructions in the [C4 docs](https://docs.c
...
+- netlify.com
+- mailgun.com
+- kickbox.com
+- moralis.io (optional)
+
+### Add environment secrets
+
+- Generate GitHub [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and set `GITHUB_TOKEN`
+- Use your GitHub username for `GITHUB_OWNER`
+- Use your email address fro `EMAIL_SENDER`
```
Comment (captainmangoC4) [inline README.md:51]: typo "for"

**Comment by Simon-Busch** [comment]
> @Simon-Busch yes, I noticed that `register-judge` was not used anywhere and was basically just a copy of `register-warden`. It seems it was replaced with `apply-for-judge`. This is probably another case of "should have done this in a separate PR" though. 🤪

Hahha :D Totally OK then ! :)

File: functions/register-team.ts [hunk 2]
```diff
@@ -102,7 +102,7 @@ exports.handler = async (event) => {
     const branchName = `team/${teamName}`;
     try {
       const res = await octokit.createPullRequest({
-        owner: "code-423n4",
+        owner: process.env.GITHUB_OWNER,
         repo: "code423n4.com",
```
Comment (joshuashort) [inline functions/register-team.ts:106]: Pull repo into environment, too?

File: functions/get-user.ts [hunk 3]
```diff
@@ -17,7 +17,7 @@ exports.handler = async (event) => {
     const user = JSON.parse(userFile.toString());
     if (user.image) {
       const imagePath = user.image.slice(2);
-      user.image = `https://raw.githubusercontent.com/code-423n4/code423n4.com/main/_data/handles/${imagePath}`;
+      user.image = `https://raw.githubusercontent.com/${process.env.GITHUB_OWNER}/code423n4.com/main/_data/handles/${imagePath}`;
```
Comment (joshuashort) [inline functions/get-user.ts:20]: If repo goes into environment, these types of paths need updated, too.

**Pull request merged**